Modules/TPC/include/TPC/CheckOfTrendings.h

```diff
@@ -41,6 +41,13 @@ class CheckOfTrendings : public o2::quality_control::checker::CheckInterface
 
  private:
   ClassDefOverride(CheckOfTrendings, 2);
+  std::string checkChoice;
+  float expectedPhysicsValue;
+  float nSigmaExpectedPhysicsValue;
+  float nSigmaBadExpectedPhysicsValue;
+  float nSigmaMean;
+  float nSigmaBadMean;
+  int pointToTakeForExpectedValueCheck;
```

Comment:
Please follow naming conventions. Data members should start with `m` and camel case.